srcpkgs/dpdk/template

```diff
@@ -0,0 +1,87 @@
+# Template file for 'dpdk'
+pkgname=dpdk
+version=19.08
+revision=1
+archs="x86_64 i686"
+build_style=meta
+hostmakedepends="cmake meson doxygen python3-Sphinx pkg-config elfutils"
+makedepends="libnuma-devel linux-headers jansson-devel libpcap-devel zlib-devel libressl-devel libbsd-devel libelf libmnl"
+depends="jansson zlib libbsd libressl python-pyelftools"
```

Comment:
`janson zlib libbsd libressl` are automatically detected.
```
SONAME: libjansson.so.4 <-> jansson>=2.4_1
SONAME: libz.so.1 <-> zlib>=1.2.3_1
SONAME: libbsd.so.0 <-> libbsd>=0.4.2_1
SONAME: libcrypto.so.45 <-> libcrypto45>=2.9.2_1
...
```

Reply:
Ok. Removed. I think I finally got it how `depends` works.